packages/rainbowkit/src/wallets/getDefaultWallets.ts

```diff
@@ -33,6 +34,7 @@ export const getDefaultWallets = ({
         coinbase({ appName, chains }),
         metaMask({ chains, shimDisconnect: true }),
         walletConnect({ chains }),
+        torus(chains),
```

Comment:
We're not going to be adding anymore wallets to `getDefaultWallets`. This function is meant as a convenience for internal development. We'd like the community to decide for themselves what wallets they'd like to support and publish packages that reflect those preferences.